app/models/daily_usage_creation/steps/time_usage.rb

```diff
@@ -14,9 +14,15 @@ class TimeUsage < WizardSteps::Step
       validates_with Validators::TimeUsageValidator
 
       def label
-        "On average, how often do you use #{added_appliance_name}(s)?"
+        "How long do you use #{added_appliance_name}(s) for?"
       end
 
+      # rubocop:disable Layout/LineLength
```

Comment:
Is it worth concatenating the string across a couple of lines so you don't have to worry about disabling the cop?

Reply:
That causes another violation (prefer string interpolation to concatenation), so I'll just leave cops disabled for this method 👍🏻